packages/cosmic-swingset/Makefile

```diff
@@ -71,8 +71,8 @@ gov-q:
 	$(AGCH) query gov proposals --output json | \
 		jq -c '.proposals[] | [.proposal_id,.voting_end_time,.status]'
 
-deploy-runStake: ../run-protocol/test/runStake/runStake-deploy.js
-	echo agoric deploy ../run-protocol/test/runStake/runStake-deploy.js
+deploy-runStake: ../inter-protocol/test/runStake/runStake-deploy.js
+	echo agoric deploy ../inter-protocol/test/runStake/runStake-deploy.js
```

Comment:
Please delete this obsolete Makefile stanza.

There is no `runStake-deploy.js` file in the repo. I suspect I had something by that name locally at the time, but I don't even have that. I suppose it's obsolete in favor of stuff like https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/src/proposals/core-proposal.js

Reply:
Oh good. `measure.js` may be similarly dead so I'll wait for an answer on that.